lib/holidays/definition/generator/regions.rb

```diff
@@ -37,8 +41,9 @@ def generate_parent_lookup(regions)
           lookup = {}
 
           regions.each do |region, subregions|
-            subregions.each do |s|
-              lookup[s] = region unless lookup.has_key?(s)
+            subregions.each do |subregion|
+              parent_region = NORTH_AMERICA_REGIONS.include?(subregion) ? subregion : region
+              lookup[subregion] = parent_region unless lookup.has_key?(subregion)
```

Comment:
Because `Rakefile` loads the "ca" definitions load first and those definitions include "north_america_informal", Canada was being defined as the parent region for the US here.